src/clients/meta/MetaClient.cpp

```diff
@@ -37,6 +39,27 @@ DEFINE_int32(meta_client_retry_interval_secs, 1, "meta client sleep interval bet
 DEFINE_int32(meta_client_timeout_ms, 60 * 1000, "meta client timeout");
 DEFINE_string(cluster_id_path, "cluster.id", "file path saved clusterId");
 DEFINE_int32(check_plan_killed_frequency, 8, "check plan killed every 1<<n times");
+DEFINE_uint32(failed_login_attempts,
+              0,
```

Comment:
what's the meaning for "0"

Reply:
See the desc of the PR.

Reply:
0 means disable this feature. I had comments in src/graph/service/GraphFlags.h